vars/notifyBuildResult.groovy

```diff
@@ -75,8 +73,6 @@ def call(Map args = [:]) {
         data['statsUrl'] = statsURL
         data['es'] = es
         data['es_secret'] = secret
```

Comment:
missing the `jobName` -> https://github.com/elastic/apm-pipeline-library/pull/1016/files#diff-f9698af828252702ef720a3af41ec8d7469294e5149a71ac4edf48c283a14834R47